src/screens/home/HomeScreen.js

```diff
@@ -0,0 +1,74 @@
+import React, { useContext, useEffect } from 'react';
+
+// Components
+import { StatusBar } from 'react-native';
+import {
+  Container,
+  Header,
+  Title,
+  Content,
+  Footer,
+  FooterTab,
+  Button,
+  Left,
+  Right,
+  Body,
+  Icon,
+  Text,
+} from 'native-base';
```

Comment:
NativeBase looks great :+1: would love to see a screenshot of what the app looks like now.

Reply:
Here's a picture of the settings page:
![image](https://user-images.githubusercontent.com/27874045/88678261-421f5480-d0bc-11ea-85ae-bf8485762193.png)

I haven't spent much time yet on the visuals but I don't plan to do too much on the setting page anyway so this may be nearly a finished page!